### PR TITLE
Use python logging

### PR DESCRIPTION
**Pull request by KKoukiou**
Replace prints with python logging.
Add two more options, one to redirect output to file and one more to increase verbosity by request.

### CODE REVIEW

File: log_extractor/log_extractor.py [hunk 1]
```diff
@@ -493,7 +497,18 @@ def parse_logs(self):
         "local compressed file with the logs."
     )
 )
-def run(job, build, folder, logs, team, skip_download, local_log_file):
+@click.option(
+    "--log-output",
+    help=(
```
Comment (cynepco3hahue) [inline log_extractor/log_extractor.py]: one line

File: log_extractor/log_extractor.py [hunk 2]
```diff
@@ -105,7 +109,7 @@ def extract_all(self, path):
                     os.path.getsize(f_path) != 0
                 ):
                     dst_path = os.path.splitext(f_path)[0]
-                    print '==== Unpack the file %s ====' % f_path
+                    logger.info('==== Unpack the file %s ====' % f_path)
```
Comment (cynepco3hahue) [inline log_extractor/log_extractor.py]: Can you on the way change it to
```
logger.info('==== Unpack the file {0} ===='.format(f_path))
```
just for consistentence

**Comment by cynepco3hahue** [comment]
lgtm, just rebase and if you in the good mood fix small comments :smile:

**Review by cynepco3hahue — APPROVED** [review]

**Comment by KKoukiou** [comment]
@cynepco3hahue should I do something extra here?

**Comment by cynepco3hahue** [comment]
@KKoukiou lgtm

**Pull request merged**